src/prefect/utilities/callables.py

```diff
@@ -176,6 +184,16 @@ class ModelConfig:
     schema = model.schema(by_alias=True)
     return ParameterSchema(**schema)
 
+def get_default_values_from_parameter_schema_dict(schema: Dict[str, Any]) -> Dict[str, Any]:
+    """Given a parameter schema, return a dictionary of the default values for each
+    parameter."""
+    values = {}
+    properties = schema.get("properties", {})
+    for key in properties:
+        values[key] = properties.get(key).get('default', None)
```

Comment:
Per the earlier note about the semantic deference between `None` and empty — we should not pass values for parameters without a value; this is distinct from passing a value of `None`.